wagtail/migrations/0075_populate_latest_revision_and_revision_title.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.0.3 on 2022-05-26 13:58
+
+from django.db import migrations, models
+
+
+def populate_latest_revision(apps, schema_editor):
+    Page = apps.get_model("wagtailcore.Page")
+    Revision = apps.get_model("wagtailcore.Revision")
+    for page in Page.objects.all():
```

Comment:
This migration is likely to take a very long time for large sites, as it requires two queries per page to find the latest revision and set the field. Some sites have over 100k pages, and database round-trips can take a lot longer on a production site than on a local machine.

Some ideas for how to optimise this:
 - Add ``.iterator()`` so we don't pull all page objects into memory at once
 - Add ``.only('content_type_id', 'id')`` to minimise the amount of data we need to fetch from the database
 - Is it possible to use a subquery to fetch the ID of the latest revision along with the pages? (For example: ``Page.objects.all().annotate(latest_revision_id=...)``)
 - Could we bulk update the ``latest_revision_id`` field?

Reply:
Thanks! I've updated this to use `.update()` with a `Subquery`.